eslint.config.js

```diff
@@ -0,0 +1,25 @@
+const js = require('@eslint/js')
+const neostandard = require('neostandard')
+const node = require('eslint-plugin-n')
+const promise = require('eslint-plugin-promise')
+const markdown = require('eslint-plugin-markdown')
+
+module.exports = [
+  ...neostandard({
```

Comment:
I had to migrate to neostandard because standard [does not support](https://github.com/standard/eslint-config-standard/issues/410#issuecomment-2138082851) eslint 9

Reply:
cc: @ctcpip @wesleytodd I know that there is long discussion on going about linters, can you confirm if we are align with your proposal for the "near future"? :pray: